### PR TITLE
[#1234] Make repo field longer

### PR DESCRIPTION
**Pull request by sujaymansingh**
For rtfd/readthedocs.org#1234

### CODE REVIEW

**Comment by ericholscher** [comment]
I'm +1 on this, but will wait to merge it until this weekend, so that I can run the migrations in production.

**Comment by hbetts** [comment]
@webmaven good eye.

Here's the actual differences in the migration files:

```
@@ -8,16 +8,14 @@ from django.db import models
 class Migration(SchemaMigration):
 
     def forwards(self, orm):
-        
-        # Changing field 'Project.repo'
-        db.alter_column('projects_project', 'repo', self.gf('django.db.models.fields.CharField')(max_length=255))
 
+        # Changing field 'Project.repo'
+        db.alter_column(u'projects_project', 'repo', self.gf('django.db.models.fields.CharField')(max_length=255))
 
     def backwards(self, orm):
-        
-        # Changing field 'Project.repo'
-        db.alter_column('projects_project', 'repo', self.gf('django.db.models.fields.CharField')(max_length=100))
 
+        # Changing field 'Project.repo'
+        db.alter_column(u'projects_project', 'repo', self.gf('django.db.models.fields.CharField')(max_length=100))
 
     models = {
         u'auth.group': {
@@ -86,7 +84,7 @@ class Migration(SchemaMigration):
             'path': ('django.db.models.fields.CharField', [], {'max_length': '255'}),
             'project': ('django.db.models.fields.related.ForeignKey', [], {'related_name': "'imported_files'", 'to': u"orm['projects.Project']"}),
             'slug': ('django.db.models.fields.SlugField', [], {'max_length': '50'}),
-            'version': ('django.db.models.fields.related.ForeignKey', [], {'related_name': "'imported_filed'", 'null': 'True', 'to': u"orm['builds.Version']"})
+            'version': ('django.db.models.fields.related.ForeignKey', [], {'related_name': "'imported_files'", 'null': 'True', 'to': u"orm['builds.Version']"})
         },
         u'projects.project': {
             'Meta': {'ordering': "('slug',)", 'object_name': 'Project'},
@@ -146,4 +144,4 @@ class Migration(SchemaMigration):
         }
     }
```

The _added/changed_ lines are from !1218

File: readthedocs/projects/migrations/0044_make_repo_longer.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        u'projects.importedfile': {
+            'Meta': {'object_name': 'ImportedFile'},
+            'commit': ('django.db.models.fields.CharField', [], {'max_length': '255'}),
+            u'id': ('django.db.models.fields.AutoField', [], {'primary_key': 'True'}),
+            'md5': ('django.db.models.fields.CharField', [], {'max_length': '255'}),
+            'name': ('django.db.models.fields.CharField', [], {'max_length': '255'}),
+            'path': ('django.db.models.fields.CharField', [], {'max_length': '255'}),
+            'project': ('django.db.models.fields.related.ForeignKey', [], {'related_name': "'imported_files'", 'to': u"orm['projects.Project']"}),
+            'slug': ('django.db.models.fields.SlugField', [], {'max_length': '50'}),
+            'version': ('django.db.models.fields.related.ForeignKey', [], {'related_name': "'imported_files'", 'null': 'True', 'to': u"orm['builds.Version']"})
```
Comment (hbetts) [inline readthedocs/projects/migrations/0044_make_repo_longer.py]: I think this should actually be `imported_filed` in place of `imported_files`, based on a quick glance through the other migrations files.

**Comment by ericholscher** [comment]
Yup, merged the other one.

**Pull request closed without merging**